client/packages/invoices/src/StockOut/Components/AutoAllocateIssueField.tsx

```diff
@@ -27,29 +27,18 @@ export const AutoAllocateField = ({
     allocatedQuantity: getAllocatedQuantity(state),
     allocateIn: state.allocateIn,
   }));
-
-  // Using buffer state with the allocated quantity, so gets pre-populated with existing
-  // quantity, and updated when the user edits the individual lines
   const [issueQuantity, setIssueQuantity] = useBufferState(allocatedQuantity);
 
-  // using a debounced value for the allocation. In the scenario where
-  // you have only pack sizes > 1 available, and try to type a quantity which starts with 1
-  // e.g. 10, 12, 100.. then the allocation rounds the 1 up immediately to the available
-  // pack size which stops you entering the required quantity.
-  // See https://github.com/msupply-foundation/open-msupply/issues/2727
-  // and https://github.com/msupply-foundation/open-msupply/issues/3532
```

Comment:
The comments makes sense. But the code seems to be straight forward enough to understand without the comment?